scripts/e2e/bootstrap_job/spec/machines.template

```diff
@@ -6,7 +6,6 @@ items:
   metadata:
     generateName: ${TARGET_VM_PREFIX}-
     labels:
-      set: master
```

Comment:
need to follow up on the discussion, so no master label required anymore?  how controller know it is master?

Reply:
this does not affect the PR merge as long as ci passed, i will dig into the code to understand it better later, thx